index.html

```diff
@@ -44,99 +45,63 @@ <h1>I'm Alexandra <br>Glad to see you!</h1>
+                    <div class="pop-up-description">
+                        <div>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <div class="pop-up-technologies">
+                            <ul class="categories">
+                                <li class="tag">html</li>
+                                <li class="tag">css</li>
+                                <li class="tag">javascript</li>
+                                <li class="tag">javascript</li>
+                                <li class="tag">javascript</li>
+                            </ul>
+                            <hr>
+                            <div class="pop-up-btns">
+                                <button type="button" class="btn">See Live <img src="images/live-icon.svg"
+                                        alt=""></button>
+                                <button type="button" class="btn">See Source <img src="images/github-blue-icon.svg"
+                                        alt=""></button>
```

Comment:
- [ ] good job so far 👏  In addition to what you have done, I would like to remind you that this section should be rendered dynamically. All the project cards should be rendered dynamically. This means that all the HTML codes concerning the works section will be removed. However, here I can see that there are still some HTML codes in the works section. Please remove these codes and make sure that this section is rendered dynamically with the help of javascript 😃 👍